lib/l10n/tr_nums.dart

```diff
@@ -0,0 +1,7 @@
+import 'dart:io';
+import 'package:intl/intl.dart';
+
+String trNums(dynamic num) {
+  final formatter = NumberFormat('###.0#', Platform.localeName);
```

Comment:
```suggestion
String trNumber(num number) {
  final formatter = NumberFormat(number.truncateToDouble() == number ? '###' : '###.0#', Platform.localeName);
```

use `num` type to force passing numbers to the function and the check to remove the zero after decimal if there's no mantissa